roles/setup_assisted_installer/tasks/main.yml

```diff
@@ -1,12 +1,43 @@
 ---
-- name: Setup AI
+- name: Setup AI Host
   become: True
   block:
     - name: Install podman
       package:
         name: podman
         state: present
 
+    - name: Configure registry cert and pull_secret
+      block:
+        - name: Copy cert to pki directory
+          copy:
+            content: "{{ mirror_certificate }}"
+            dest: /etc/pki/ca-trust/source/anchors/ai_registry.crt
+            mode: 0660
+            force: yes
+            backup: yes
+
+        - name: Update the CA trust files
+          command: update-ca-trust extract
+
+        - name: Create podman auth dir
+          file:
+            path: "{{ config_file_path }}/containers/"
+            state: directory
+            mode: 0755
+
+        - name: Copy pull_secrets file.
+          copy:
+            content: "{{ pull_secret }}"
+            dest: "{{ config_file_path }}/containers/auth.json"
+            mode: 0644
+            remote_src: yes
+  
+  when: use_local_mirror_registry | bool
```

Comment:
Podman installation is now bounded with the local_mirror_registry flag. Is that what we want here?

Reply:
Good spot that is at the wrong indent :)